### PR TITLE
Consistent rust version in ci

### PR DESCRIPTION
**Pull request by realbigsean**
## Issue Addressed

N/A

## Proposed Changes

`rustup update` in every step

### CODE REVIEW

**Review by michaelsproul — APPROVED** [review]
Looks good to me, I think we should probably merge it to unblock all the other PRs.

**Review by paulhauner — APPROVED** [review]
Nice one! Thanks @realbigsean!

bors r+

File: .github/workflows/test-suite.yml [hunk 1]
```diff
@@ -106,6 +114,8 @@ jobs:
     needs: cargo-fmt
     steps:
       - uses: actions/checkout@v1
+      - name: Get latest version of stable Rust
+      run: rustup update stable
```
Comment (michaelsproul) [inline .github/workflows/test-suite.yml]: I wonder if this indentation is causing issues?

**Comment by michaelsproul** [comment]
thanks @realbigsean, @paulhauner and I rolled this into #2193

**Pull request closed without merging**